inst/shiny/ui_01_gene_sets.R

```diff
@@ -72,9 +72,9 @@ shinyPanelGeneSets <- fluidPage(
 
     selectInput("gsByParam", "Location within SCE object where the gene identifiers in should be mapped.", list()),
 
-    withBusyIndicatorUI(
-      actionButton("uploadGS", "Upload")
-    ),
+    #withBusyIndicatorUI(
```

Comment:
I think it is okay to delete these lines instead of commenting to keep the code clean